peacebot/core/plugins/Moderation/mod.py

```diff
@@ -0,0 +1,259 @@
+from datetime import datetime, timedelta, timezone
+
+import hikari
+import lightbulb
+from lightbulb.utils import nav
+
+from models import ModLogs
+from peacebot.core.utils.embed_colors import EmbedColors
+from peacebot.core.utils.errors import ModerationError
+from peacebot.core.utils.helper_functions import convert_time
+from peacebot.core.utils.permissions import (
+    higher_role_check,
+    mod_logs_check,
+    mod_role_check,
+    moderation_role_check,
+    register_cases,
+)
+from peacebot.core.utils.utilities import _chunk
+
+mod_plugin = lightbulb.Plugin("Mod", "Commands to be used by the Moderators")
+
+
+@mod_plugin.command
+@lightbulb.command("timeout", "Group that consists of the timeout commands")
+@lightbulb.implements(lightbulb.SlashCommandGroup, lightbulb.PrefixCommandGroup)
+@moderation_role_check
+async def timeout(ctx: lightbulb.Context) -> None:
+    await ctx.bot.help_command.send_group_help(ctx, ctx.command)
+
+
+@timeout.child
+@lightbulb.option("reason", "Reason for the timeout", type=str)
+@lightbulb.option("time", "The duration of timeout (Eg. 10s, 3d, 5d)", type=str)
+@lightbulb.option("member", "The member to timeout", type=hikari.Member)
+@lightbulb.command("enable", "Timeout a member from the server")
+@lightbulb.implements(lightbulb.SlashSubCommand, lightbulb.PrefixSubCommand)
+@moderation_role_check
+async def timeout_enable_command(ctx: lightbulb.Context) -> None:
```

Comment:
The code is clean even though it's multiline, it's uses helper and reusable functions so it's good.

But you aren't leaving any spacing at all, It'd be better if you leave spaced after 44 or 46, so that it's clear for the code viewer and the code looks clean.

Reply:
Will do!